core/src/main/java/cucumber/runtime/arquillian/ArquillianCucumber.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Overall note: seems indentation + style was broken (some { are not "on the same line") etc, it should be clean up